encoders/image/ImageOnnxEncoder/manifest.yml

```diff
@@ -7,7 +7,7 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
```

Comment:
The model is loaded from https://github.com/onnx/models.git in the Dockerfile. 
It got recently updated and does not work with our tests anymore.
You can fix the version to the previous one in the Dockerfile. 
`git checkout cbda9ebd037241c6c6a0826971741d5532af8fa4`

Reply:
can it be found elsewhere?

Reply:
Done